modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java

```diff
@@ -329,6 +351,33 @@ private void parseMetaData() {
         return underlying;
     }
 
+    /**
+     * Create a copy of metadata
+     *
+     * @param originalMetadata          source metadata that needs to be copied
+     * @param newCompressedMessageValue Value to indicate if the message is compressed or not
+     * @return copy of the metadata as a byte array
+     */
+    private byte[] createNewMetadata(byte[] originalMetadata, boolean newCompressedMessageValue) {
+        ByteBuffer buf = ByteBuffer.wrap(originalMetadata);
+        buf.position(1);
+        buf = buf.slice();
+        MessageMetaDataType type = MessageMetaDataType.values()[originalMetadata[0]];
+        metaDataType = type;
+        StorableMessageMetaData original_mdt = type.getFactory()
```

Comment:
As a practice  '_' should be used only in constants and shouldn't be used anywhere else in the code.